src/rebar_httpc_adapter.erl

```diff
@@ -0,0 +1,41 @@
+%% Derived from hex_core v0.7.1 for extra flexibility.
```

Comment:
You should use the v0.5.3 comment for this I believe: https://github.com/erlang/rebar3/blob/v3.13/src/r3_hex_http_httpc.erl -- the file is the same across versions but we should be consistent.

Reply:
`v0.5.3` or `v0.5.1`? I went with the version in the link, but happy to patch again if it really should be `v0.5.3`.